CONTRIBUTING.md

```diff
@@ -49,7 +125,31 @@ However, there are still some styles that the linter cannot pick up, notably in
 * If a method header doesn't fit on one line each argument goes on a separate line.
 * 100 character line length
 
-## License
+## Tests
+
+Tests help us prevent regressions from being introduced to the codebase. The GitHub repository is continuously tested using Circle and Appveyor, the results of which are available through the Checks functionality on [commits](https://github.com/facebook/react-native/commits/master) and pull requests. You can learn more about running and writing tests in the [Tests wiki](http://github.com/facebook/react-native/wiki/Tests).
```

Comment:
Should this link to the nw testing doc as well?

Reply:
Which one is that?

Reply:
Nvm, I see that's the wiki/Tests.md linked here 👌